### ISSUE TITLE
Checkinstall only aviable in Debian 10 BAckports

### ISSUE THREAD

**Issue by gOOvER** [SoftCreatR/imei#35]
Install fails, because checkinstall not found., It's not in an Debian 10, only in Backports

https://packages.debian.org/buster-backports/checkinstall

log:

```
Paketlisten werden gelesen...
Abhängigkeitsbaum wird aufgebaut....
Statusinformationen werden eingelesen....
E: Paket checkinstall kann nicht gefunden werden.
```

`/dev/fd/63: Zeile 476: checkinstall: Kommando nicht gefunden.`

**Comment by SoftCreatR** [comment]
Should be fixed via e73352b930cd6c5e29c5a3c63dd8c07a4e6517bd

**Comment by SoftCreatR** [comment]
There might be issues using Checkinstall on Debian. In this case, uninstall any packages generated by IMEI:

`apt remove imei-* --purge`

And run IMEI again using `--make`. By doing so, Checkinstall won't be used at all and everything should work, as it did before.
